cmd/manifestd/cmd/commands.go

```
@@ -5,6 +5,7 @@
 	"io"
 	"os"
 
+	"github.com/CosmWasm/wasmd/x/wasm"
```

golangci-lint (1.62.2)

File is not `gci`-ed with --skip-generated -s standard -s default -s blank -s dot -s prefix(github.com/cometbft/cometbft) -s prefix(github.com/cosmos) -s prefix(cosmossdk.io) -s prefix(github.com/cosmos/cosmos-sdk) -s prefix(github.com/strangelove-ventures/poa) -s prefix(github.com/strangelove-ventures/tokenfactory) -s prefix(github.com/CosmWasm/wasmd) -s prefix(github.com/CosmWasm/wasmvm) -s prefix(github.com/liftedinit/manifest-ledger) --custom-order

(gci)